extensions/wikia/Recirculation/templates/Recirculation_wikiArticles.mustache

```diff
@@ -0,0 +1,25 @@
+<div class="mcf-card mcf-card-wiki-articles">
+	<header class="mcf-card-wiki-articles__header"{{#communityHeaderBackground}} style="background-image: url({{communityHeaderBackground}});"{{/communityHeaderBackground}}>
+	<span class="mcf-card-wiki-articles__header-text">More {{sitename}}</span>
+	</header>
+	<ul class="mcf-card-wiki-articles__list">
+		<li class="mcf-card-wiki-articles__item">
+			<a href="#" class="mcf-card-wiki-articles__item-link">
+				<span class="mcf-card-wiki-articles__circle">1</span>
```

Comment:
could this be done with css counters?
https://stackoverflow.com/questions/23610151/can-you-style-ordered-list-numbers